pygmyui/static/js/app.js

```diff
@@ -81,15 +81,22 @@ function CheckLinkAvailability(){
 }
 
 function DeleteLink(el){
-    var id = el.dataset.id.split("/").pop();
-    
-    $.ajax({
-        type: "GET",
-        url: '/remove',
-        data: {link_id: id},
-        dataType: 'json',
-        success: function(responsedata){
-            window.location.reload();
-        }
-    })
-}
\ No newline at end of file
+    // Confirmation dialog before deleting link
+    const confirmResponse = confirm('Are you sure you want to delete the link?');
+
+    // Delete link if user confirmed
+    if (confirmResponse) {
+        var id = el.dataset.id.split("/").pop();
+        
+        $.ajax({
+            type: "GET",
+            url: '/remove',
+            data: {link_id: id},
+            dataType: 'json',
+            success: function(responsedata){
+                window.location.reload();
+            }
+        })
+        alert("link deleted!");
```

Comment:
Remove alert.
Clicking once to confirm and seeing the link removed after the `window.location.reload()` should be enough for the user to see it has been deleted.